### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MattNguyen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,9 +94,9 @@ test('Return Value', function(assert) {
 });
 ```
 
-In the above example, we added a third parameter to createSpy(), which Js.Edgar used to mock a return value when foo() was called.  This parameter can be any valid javascript object or primitive, including functions!
+In the above example, we added a third parameter to `createSpy()`, which Js.Edgar used to mock a return value when `foo()` was called.  This parameter can be any valid javascript object or primitive, including functions!
 
-If you need to change the Spy's return value for subsequent assertions or to setup mocking after executing the live method previously (see below), you can call startMocking() (or its alias andMock()).
+If you need to change the Spy's return value for subsequent assertions or to setup mocking after executing the live method previously (see below), you can call `startMocking()` (or its alias `andMock()``).
```
Comment (elwayman02) [inline README.md]: Why two `` here?

Comment (MattNguyen) [inline README.md]: :dancer:

**Comment by MattNguyen** [comment]
@elwayman02 updated

**Comment by elwayman02** [comment]
Thanks! :shipit:

**Pull request merged**